mppsolar/__init__.py

```diff
@@ -152,6 +152,14 @@ def main():
         help="Mongo db name (default: mppsolar)",
         default="mppsolar",
     )
+    parser.add_argument(
```

Comment:
the need for additional config parameters is one of the things Im trying to address in the powermon version (using a config yaml to allow different and flexibile config for each item)